tests/comments/test_async_templatetags.py

```diff
@@ -0,0 +1,185 @@
+import html
+import json
+import re
+
+import pytest
+from django.contrib.auth.models import AnonymousUser
+from django.contrib.contenttypes.models import ContentType
+from django.urls import reverse
+from freezegun import freeze_time
+
+from adhocracy4.test import helpers
+from tests.apps.questions.phases import AskPhase
+
+
+def react_comment_render_for_props(rf, user, question):
```

Comment:
I was wondering that before: where does this rf fixture come from and what does it stand for actually? :sweat_smile:

Reply:
That is a very good question! I think it stands for RequestFactory, but I have no idea how it gets into our tests as rf.
https://docs.djangoproject.com/en/3.2/topics/testing/advanced/#the-request-factory
@Rineee

Reply:
Ah cool, thanks! Now that I knew what to look for, I investigated and it comes with pytest, see here: https://github.com/pytest-dev/pytest-django/blob/6ac1495c17d94db1293659edb01966c278064114/pytest_django/fixtures.py#L378

Reply:
Nice! Thanks!